Code/Framework/AzFramework/AzFramework/Asset/AssetProcessorMessages.h

```diff
@@ -1254,6 +1254,44 @@ namespace AzFramework
             FolderList m_folderList;
         };
 
+        class AssetChangeReportRequest
+            : public BaseAssetProcessorMessage
+        {
+        public:
+            AZ_CLASS_ALLOCATOR(AssetChangeReportRequest, AZ::OSAllocator, 0);
+            AZ_RTTI(AssetChangeReportRequest, "{EF108E73-08F7-4CBC-A808-61A2EC544A6E}", BaseAssetProcessorMessage);
+            static void Reflect(AZ::ReflectContext* context);
+            static constexpr unsigned int MessageType = AZ_CRC_CE("AssetSystem::AssetChangeReport");
+
+            enum ChangeType
+            {
+                CheckMove,
+                Move
+            };
+
+            AssetChangeReportRequest() = default;
```

Comment:
Nitpick: if this is only required for the SerializeContext and not intended for use then a comment saying something like that could be handy.

Reply:
Done.